### ISSUE TITLE
question about the pretrained model

### ISSUE THREAD

**Issue by glynpu** [hellochick/ICNet-tensorflow#21]
The dataset has 19 classes, plus the background,  the output of last layer should be 20 classes. But the pre-trained model produces a 19 channels output. Do the background just be ignored? But in the train code, the ignored-label is set to 0.    
I am a little confused, please help!

**Comment by hellochick** [comment]
Hey, sorry for confusing you. The ignore-label of cityscapes is not 0 (is 255), I forgot to change it before I upload the training code. So when training, we will focus on the 0~18 classes and ignore the background (255), which means we don't want to learn how to recognize the background. Does that make sense to you?
